presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -423,6 +425,11 @@ public SystemSessionProperties(
                         "Use resources which are not guaranteed to be available to the query",
                         false,
                         false),
+                dataSizeProperty(
+                        QUERY_MAX_SCAN_PHYSICAL_BYTES,
+                        "Maximum scan physical bytes of a query",
```

Comment:
Let's clarify. Is this before or after compression? Before or after filtering? I assume it is after compression but before filtering.

Reply:
Thank you Maria, this is a good point. So to clarify the confusion how the scan size is measured, I have implicitly implemented the SqlQueryExecution.getScannedBytes method to extract the query scanned size from QueryInfo's rawInputDataSize stats which is the consumed input bytes, and yes it should be the actual data size before filtering.

Reply:
Should we just say `QUERY_MAX_SCAN_RAW_INPUT_BYTES` that way it matches the public stat that we publish and it is easy for people to understand which metric did they exceed? @mbasmanova thoughts?